src/components/UpperFooter/UpperFooterList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const UpperFooterList = props => {
+  const { title, list } = props;
+
+  return (
+    <Ul>
+      <ListTitle>{title}</ListTitle>
+      {list.map(el => {
+        return <Li key={el.id}>{el.name}</Li>;
+      })}
+    </Ul>
+  );
+};
+
+export default UpperFooterList;
+
+const Ul = styled.ul`
```

Comment:
```suggestion
const Container = styled.ul`
```
저는 그냥 가장 바깥 쪽 요소는 고민 없이 container라고 짓는 편입니다

Reply:
참고하여 전반적으로 가장 바깥쪽 컴포넌트명은 Container로 변경하였습니다.